pkg/controller/security/security.go

```diff
@@ -0,0 +1,37 @@
+package security
```

Comment:
nit: Do these need to be defined as global package variables given they're unexported?